test/opsrc/basic_test.go

```diff
@@ -0,0 +1,79 @@
+package opsrc
```

Comment:
please use `operatorsource` as the package name

Reply:
I renamed the package.